### PR TITLE
Feature/oui sidebar

### PR DESCRIPTION
**Pull request by AxelPeter**
- Update sidebar to new style
- Update dependencies

### CODE REVIEW

File: src/components/showcase/showcase.html [hunk 1]
```diff
@@ -12,34 +12,39 @@
...
-                    </li>
-                </ul>
+            <nav class="oui-sidebar oui-showcase__sidenav" role="complementary">
+                <div class="oui-sidebar-menu">
+                    <ul class="oui-sidebar-list">
+                        <li class="oui-sidebar-list__item"
+                            ng-repeat="groupKey in $ctrl.secondLevelsChildren[$ctrl.currentSecondLevelStateName].groupsOrder track by $index"
+                            ui-sref-active="oui-sidebar-list__item_active">
+                            <button class="oui-sidebar-link" type="button"
+                                ng-if="groupKey === 'undefined'"
```
Comment (FredericEspiau) [inline src/components/showcase/showcase.html]: `groupKey === undefined` ?

Comment (AxelPeter) [reply src/components/showcase/showcase.html]: `groupKey === "undefined"`, not really the same.
But I'll check if that cannot be simplified (legacy code)

**Comment by ovh-cds** [comment]
CDS Report yarn-build#138.0 ✘  
* build &amp; test
  * build ✘  
  * test ✔

**Comment by ovh-cds** [comment]
CDS Report yarn-build#139.0 ✘  
* build &amp; test
  * build ✘  
  * test ✔

**Pull request merged**